ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js

```diff
@@ -222,17 +222,20 @@
       netRegex: NetRegexes.headMarker({ id: ['004D', '004E'] }),
       condition: Conditions.targetIsYou(),
       delaySeconds: 0.5,
-      infoText: function(data, matches) {
+      infoText: function(data, matches, output) {
         const partner = matches.id === '004D' ? '004E' : '004D';
         // If for some reason there is no partner, we get a vulnerability or bleed and are sad.
         if (!data[partner])
           return;
-        return {
-          en: 'Stack with ' + data.ShortName(data[partner]),
-          de: 'Sammeln mit ' + data.ShortName(data[partner]),
-          fr: 'Packez-vous avec ' + data.ShortName(data[partner]),
-          cn: '靠近' + data.ShortName(data[partner]) + '集合',
-        };
+        return output.text({ player: data.ShortName(data[partner]) });
```

Comment:
nit: `partner`?

Reply:
I think I'm going to stick with `${player}` for any player names, for consistency.